bindata/network/multus/000-ns.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: multus
+  labels:
+    name: multus
+    openshift.io/run-level: "0"
```

Comment:
does multus actually need this?

Reply:
Yes, definitely - to be except from SecurityContextConstraints so it can create privileged pods.